plenum/server/replica.py

```diff
@@ -809,7 +820,7 @@ def create3PCBatch(self, ledger_id):
             pp_seq_no,
             tm,
             [req.digest for req in reqs],
-            len(validReqs),
+            self._unpack_discarded_mask(discarded_mask),
```

Comment:
Can we do an optimization so that if there are no discarded requests (all passed dynamic validation), then we just use an empty (None or zero) mask.